src/test/extension.test.ts

```diff
@@ -44,4 +36,35 @@ suite('DeviceTree Extension Test Suite', () => {
             assert.strictEqual(doc.languageId, 'dts');
         });
     });
+
+    suite('Formatter Tests', () => {
```

Comment:
I recommend moving formatting tests into a separate file.
Also, I would not use an external file as input for your tests. Instead, use multiline strings, which you can define directly in the unit test. This way, you can have more unit tests, each testing only a specific part of what the formatter should do.

Reply:
Looks much better now. I'm resolving this comment and will add some more specific ones.